net-analyzer/openbsd-netcat/openbsd-netcat-1.190.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
the whole package should be probably re-named to match the upstream `netcat-openbsd` name

Reply:
@Zlogene I have chosen to name the package as `openbsd-netcat` for consistency.  Because we have another package `net-analyzer/gnu-netcat`.

Considering this, there seems to be no dominating superiority between the 2 names.  So I would like to keep the status quo.